src/demo/app/app.module.ts

```diff
@@ -8,9 +8,9 @@ import { DemoComponent } from './demo.component';
 import { AppRoutingModule } from './app-routing.module';
 
```

Comment:
You are not making meaningful changes here.  It is only formatting changes due to your linter being setup differently.  Setup your linter to be like the project so that we will not conflict.  Then set this file back to it's original state and recommit.